And while you're doing CEF things, I get the following error:
```
CMake Error at build/cmake/setup.cmake:617 (string):
  string no output variable specified
Call Stack (most recent call first):
  build/cmake/main.cmake:42 (include)
  CMakeLists.txt:100 (include)


CMake Error at build/cmake/setup.cmake:618 (if):
  if given arguments:

    "STREQUAL" "DEBUG"

  Unknown arguments specified
```

Because `CMAKE_BUILD_TYPE` does not exist with msvc generator, it uses `CMAKE_CONFIGURATION_TYPES`. Can you fix it by using:
```cmake
if(wxUSE_WEBVIEW_CHROMIUM AND DEFINED CMAKE_BUILD_TYPE)
```
Thanks

--- a/build/cmake/init.cmake
+++ b/build/cmake/init.cmake
@@ -506,11 +506,11 @@ if(wxUSE_GUI)
             message(WARNING "WebviewChromium libcef_dll_wrapper can only be built with MSVC... disabled")
             wx_option_force_value(wxUSE_WEBVIEW_CHROMIUM OFF)
         endif()
-        if(wxUSE_WEBVIEW_CHROMIUM AND CMAKE_CXX_STANDARD LESS 14)
+        if(wxUSE_WEBVIEW_CHROMIUM AND CMAKE_CXX_STANDARD LESS 17)
             # We shouldn't disable this option as it's disabled by default and
             # if it is on, it means that CEF is meant to be used, but we can't
             # continue neither as libcef_dll_wrapper will fail to build.
-            message(FATAL_ERROR "WebviewChromium requires at least C++14 but configured to use C++${CMAKE_CXX_STANDARD}")
+            message(FATAL_ERROR "WebviewChromium requires at least C++17 but configured to use C++${CMAKE_CXX_STANDARD}")
         endif()
     endif()
 

--- a/build/cmake/lib/webview_chromium/CMakeLists.txt
+++ b/build/cmake/lib/webview_chromium/CMakeLists.txt
@@ -59,7 +59,6 @@ set(USE_SANDBOX OFF)    # Disable usage of sandbox on windows
 add_subdirectory(${CEF_ROOT} ${CEF_BUILD_DIR} EXCLUDE_FROM_ALL)
 set_target_properties(libcef_dll_wrapper PROPERTIES
     FOLDER "Third Party Libraries"
-    COMPILE_FLAGS     "-Wno-extra"
     OUTPUT_NAME "libcef_dll_wrapper"
 )
 
@@ -68,6 +67,8 @@ if(MSVC)
     get_target_property(cef_compile_options libcef_dll_wrapper COMPILE_OPTIONS)
     string(REGEX REPLACE "/MTd?;" "" cef_compile_options "${cef_compile_options}")
     set_target_properties(libcef_dll_wrapper PROPERTIES COMPILE_OPTIONS "${cef_compile_options}")
+else()
+    set_target_properties(libcef_dll_wrapper PROPERTIES COMPILE_OPTIONS "-Wno-extra")
 endif()
 
 add_library(libcef SHARED IMPORTED GLOBAL)

--- a/build/cmake/setup.cmake
+++ b/build/cmake/setup.cmake
@@ -613,7 +613,8 @@ check_type_size(ssize_t SSIZE_T)
 
 test_big_endian(WORDS_BIGENDIAN)
 
-if(wxUSE_WEBVIEW_CHROMIUM)
+# For generators using build type, ensure that wxHAVE_CEF_DEBUG matches it.
+if(wxUSE_WEBVIEW_CHROMIUM AND DEFINED CMAKE_BUILD_TYPE)
     string(TOUPPER ${CMAKE_BUILD_TYPE} build_type)
     if(${build_type} STREQUAL DEBUG)
         set(wxHAVE_CEF_DEBUG ON)

